modules/util/client-extension-util-spring-boot/src/main/java/com/liferay/client/extension/util/spring/boot/LiferayOAuth2ResourceServerEnableWebSecurity.java

```diff
@@ -202,6 +216,9 @@ private List<String> _getAllowedOrigins() {
 	@Value("${com.liferay.lxc.dxp.server.protocol}")
 	private String _lxcDXPServerProtocol;
 
+	@Value("${server.public.endpoints:}")
```

Comment:
is the trailing `:` in this correct?  I guess I'd like to see this be specified in this file in the sample: https://github.com/liferay/liferay-portal/blob/master/workspaces/liferay-sample-workspace/client-extensions/liferay-sample-etc-spring-boot/src/main/resources/application-default.properties